src/ol/source/Cluster.js

```diff
@@ -180,6 +198,28 @@ class Cluster extends VectorSource {
     this.updateDistance(this.distance, minDistance);
   }
 
+  /**
+   * Current function used to create cluster features
+   * @return {function(Point, Array<Feature>):Feature | undefined} cluster creation function
+   * @api
+   */
+  getCreateClusterFeature() {
+    return this.createClusterFeature;
```

Comment:
If you provide this getter, it should also return the default if no custom function is configured.

Reply:
I'm not sure if it should return default. This way we can check if any is provided, or default is used. If we return the default, we'll need to check if it's the one we want or not (or if it's default)

Reply:
If you export the default function, you can easily check if it's the default:
```js
if (cluster.getCreateClusterFeature() === defaultCreateClusterFeature)
```
But I'd be inclined to not have a `getCreateClusterFeature()` getter at all, and maybe not even a setter. What would be a use case for needing those? It's cheap to create a new cluster source.